### PR TITLE
[SD-1234]: Closing an item should return user to previous desk/search vi...

### PR DESCRIPTION
**Pull request by sjunaid**
...ew

### CODE REVIEW

File: app/scripts/superdesk/menu/notifications/views/notifications.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="notification-pane" ng-class="{show: flags.notifications}">
+<div ng-click="setReferrer()" class="notification-pane" ng-class="{show: flags.notifications}">
```
Comment (petrjasek) [inline app/scripts/superdesk/menu/notifications/views/notifications.html]: why calling it on click?

Comment (petrjasek) [inline app/scripts/superdesk/menu/notifications/views/notifications.html]: @sjunaid

Comment (sjunaid) [inline app/scripts/superdesk/menu/notifications/views/notifications.html]: @petrjasek , I have sent the New PR for SD-1234 which is Modified version, so ignore this change. this is no longer valid.

Comment (petrjasek) [inline app/scripts/superdesk/menu/notifications/views/notifications.html]: you mean this line is not valid or the whole PR? if the PR then pls close it

**Comment by petrjasek** [comment]
so this was part of #713 right @sjunaid ?

**Comment by sjunaid** [comment]
Yes this is now #713 , ignore #663.

**Pull request closed without merging**